frontend/package.json

```
@@ -28,11 +28,12 @@
     "redux": "^4.1.2",
     "redux-persist": "^6.0.0",
     "typescript": "~4.5.4",
-    "web-vitals": "^2.1.0"
+    "web-vitals": "^2.1.0",
+    "precompress": "7.0.1"
   },
   "scripts": {
     "start": "react-scripts start",
-    "build": "react-scripts build",
+    "build": "react-scripts build && precompress -i json,js,css,xml,svg,html,ttf build",
```

`precompress` generates both `.gz` and `.br` by default. Let's keep `.gz` only:
```suggestion
    "build": "react-scripts build && precompress -t gz -i json,js,css,xml,svg,html,ttf build",
```